studies.py

```diff
@@ -316,3 +317,47 @@ def objective(trial: optuna.Trial) -> float:
     )
     study.optimize(objective, n_trials=50, timeout=600)
     return study
+
+
+def create_single_objective_studies_for_timeline() -> List[Tuple[str, StudiesType]]:
+    studies: List[Tuple[str, StudiesType]] = []
+    storage = optuna.storages.InMemoryStorage()
+
+    def objective_timeline(trial: optuna.Trial) -> float:
+        x = trial.suggest_float("x", 0, 1)
+        time.sleep(x * 0.1)
+        if x > 0.8:
+            raise ValueError()
+        if x > 0.4:
+            raise optuna.TrialPruned()
+        return x ** 2
+
+    # Single-objective study
+    study = optuna.create_study(
+        study_name="A single objective study consuming time",
+        storage=storage,
+    )
+
+    study.enqueue_trial({"x": 0.3})
+    study.enqueue_trial({"x": 0.9})
+    study.enqueue_trial({"x": 0.5})
+    study.optimize(objective_timeline, n_trials=50, n_jobs=2, catch=(ValueError,))
+    studies.append((study.study_name, study))
+
+    # Single-objective study
+    study = optuna.create_study(
+        study_name=(
+            "A single objective study consuming time and "
+            "the order of legends is different from the order of trials"
+        ),
+        storage=storage,
+    )
+    study.enqueue_trial({"x": 0.9})
+    study.enqueue_trial({"x": 0.5})
+    study.enqueue_trial({"x": 0.3})
+    study.optimize(objective_timeline, n_trials=50, n_jobs=2, catch=(ValueError,))
+    studies.append((study.study_name, study))
+
+    # No trials single-objective study
+    optuna.create_study(study_name="A single objective study that has no trials", storage=storage)
```

Comment:
This study was not added to `studies`. Do you intentionally exclude it?

Reply:
No, it is a mistake. I have fixed the code to include the study on line#362.

Reply:
Thanks!